### PR TITLE
Updated channel link

### LINKED ISSUE #183: CircuitVerse Discord
#### Describe your bug/feature -

I want to startup an unnofficial CircuitVerse Fan Discord Server, as an alternative means of community connections. I wanted to first request permission to do this, to avoid any conflicts with other plans.

#### Current behaviour -

There is no discord.

#### Expected behaviour -

There is a discord.

#### Are you working on this issue? (Yes/No)

Yes

### PR DESCRIPTION
**Pull request by creyente1897**
Fixes #183 

#### Describe the changes you have made in this pr -
Added community channel links to the readme file

### CODE REVIEW

**Comment by creyente1897** [comment]
@Parikshit-Hooda Can you please review this.

**Comment by Parikshit-Hooda** [comment]
Looks good to me. I will merge this for now. I will not close the issue as I have not actually examined it. Will label this appropriately.

**Comment by Parikshit-Hooda** [comment]
@creyente1897  Is this commit under GSSoC?

**Comment by creyente1897** [comment]
@Parikshit-Hooda  Yes sir

**Comment by armudgal** [comment]
@creyente1897 Don't call anyone Sir :P Next time we will deduct points 😄

File: README.md [hunk 1]
```diff
@@ -119,6 +119,15 @@ bundle exec sidekiq -e production -q default -q mailers -d -L tmp/sidekiq.log` (
...
 This repository contains the [Code of Conduct](./CODE%20OF%20CONDUCT.md) of [CircuitVerse](https://circuitverse.org) Community.
 
+## CircuitVerse Community
+
+We would love to hear from you :smile:
+Here are links to our:
+
+[Gitter Channel](https://gitter.im/CircuitVerse/community)
+
+[Discord Channel](https://discord.gg/95x8H7b)
```
Comment (armudgal) [inline README.md]: Please mention that this is an unofficial fan communication channel. We absolutely don't want people landing to discord and DMing us for bug fixes or setup problems.

Comment (armudgal) [reply README.md]: Also mention @jbox144 for the initiative

Comment (creyente1897) [reply README.md]: @armudgal Ok will do so. I think I got confused between GSoC and GSSoC. I am sorry for that

Comment (creyente1897) [reply README.md]: @armudgal I think I would have to make another pr right?

Comment (armudgal) [reply README.md]: @creyente1897 Looks like it

**Pull request merged**